### DISCUSSION TITLE
Congratulation!

### DISCUSSION THREAD

**Discussion by thirdygayares** [thisisnotsnorlax/InvetoryManagementSystem_Documentation#5] [Announcements]
Your group submitted **first** and some parts of your document are okay, but others need to be set aside for now You can choose one group activity from numbers 1 to 15, and I will add +20 points.

Also, expect that your system will collaborate with an Android app made by 3rd-year students. You will also work with backend developers and System administrators from BSIT 4B.

Reply to this discussion with the group activity you want to add.

Additional points promo expired at 11:59 PM on September 30, 2024, if you did not reply

![image](https://github.com/user-attachments/assets/37921bad-20d4-44e1-897b-5a2f7edee268)

**Comment by kelvinsoliman** [comment]
paki add sa activity 1 daw po sir, PGI yung 79 lang po. 

tapos po may concern kami sir sa grades po yung oral recitation po 94 po kami don naging 70 ayan po ata yung di kami naka pasa ng video namin nung nag present kami ng github kaya na -5 yung sabi niyo po 99. thankyou sir.

**Reply by thirdygayares** [reply]
oo kayo pala yun noted!

**Reply by thirdygayares** [reply]
oo kayo pala yun noted!

**Reply by kelvinsoliman** [reply]
thankyou ng maramiii sir

**Reply by bananacake2200** [reply]
good evening po sir @thirdygayares yung concern ko po sa may forking po na email ko na po sa inyo, ch-check mo po ba yun sir?

**Reply by thirdygayares** [reply]
@bananacake2200 ano bang pangalan mo?

**Reply by bananacake2200** [reply]
Marlyn Cureg po

**Reply by thirdygayares** [reply]
ito ba frontend niyo?


![image](https://github.com/user-attachments/assets/989a83a1-fb34-48ce-bff2-2a49a1a987a3)

**Reply by thirdygayares** [reply]
pahiwalay nalang

**Reply by kelvinsoliman** [reply]
sir ako din po, soliman nasa gmail niyo po yung gawa ko:>

**Comment by bananacake2200** [comment]
thank you po sir.

**Comment by thirdygayares** [comment]
@kibs13 sobra na ang group act #1 mo saan mo gustong ilagay

**Reply by kibs13** [reply]
kayu na po bahala sir ung may 70 po hehe

**Comment by thirdygayares** [comment]
done

**Reply by kibs13** [reply]
salamat po madami sir

**Comment by thirdygayares** [comment]
okay nayung iba

mali ka sa fork ng frontend

**Reply by bananacake2200** [reply]
opo sirrr ayusin ko nalang po, thank you po.
